xen/xenops_interface.ml

```diff
@@ -546,11 +548,27 @@ module XenopsAPI (R : RPC) = struct
         Host.guest_agent_feature_list
 
     type cpu_features_array = int64 array [@@deriving rpcty]
+    type policy_array = string [@@deriving rpcty]
```

Comment:
An array of bytes I assume just more compactly encoded?

Reply:
I'm not entirely sure I understand the comment.
It's a string here as this is what Xapi will use to convey a policy